handlebars/src/main/java/com/github/jknack/handlebars/internal/Locales.java

```diff
@@ -0,0 +1,43 @@
+package com.github.jknack.handlebars.internal;
+
+import java.util.IllformedLocaleException;
+import java.util.Locale;
+
+import org.apache.commons.lang3.LocaleUtils;
+
+/**
+ * Utility methods to resolve {@link Locale}s from Strings.
+ *
+ */
+public final class Locales {
+
+  private Locales() {
+  }
+
+  /**
+   * <p>Converts {@link String}s to {@link Locale}s. Supports both 
+   * <a href="https://en.wikipedia.org/wiki/IETF_language_tag">IETF language tags</a> 
+   * using hyphens (e.g. &quot;de-AT&quot;) and the Java format using underscores (e.g. &quot;de_AT&quot;).</p>
+   * 
+   * 
+   * @param string, either in IETF or Java format (hyphen or underscore)
+   * @return locale
+   */
+  public static final Locale fromString(String string) {
+    if (string == null) {
```

Comment:
silly LocaleUtils :(

let's keep it them, but can you please add a null test?

Thanks